crypto/src/hash/pedersen/parameters.rs

```diff
@@ -0,0 +1,72 @@
+use lambdaworks_math::{
+    elliptic_curve::{
+        short_weierstrass::{
+            curves::stark_curve::StarkCurve, point::ShortWeierstrassProjectivePoint,
+            traits::IsShortWeierstrass,
+        },
+        traits::FromAffine,
+    },
+    field::{
+        element::FieldElement, fields::fft_friendly::stark_252_prime_field::Stark252PrimeField,
+    },
+};
+
+pub struct PedersenParameters<SW: IsShortWeierstrass> {
+    pub curve_const_bits: usize,
+    pub table_size: usize,
+    pub shift_point: ShortWeierstrassProjectivePoint<SW>,
+    pub points_p1: Vec<ShortWeierstrassProjectivePoint<SW>>,
+    pub points_p2: Vec<ShortWeierstrassProjectivePoint<SW>>,
+    pub points_p3: Vec<ShortWeierstrassProjectivePoint<SW>>,
+    pub points_p4: Vec<ShortWeierstrassProjectivePoint<SW>>,
+}
+
+impl Default for PedersenParameters<StarkCurve> {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
+impl PedersenParameters<StarkCurve> {
+    pub fn new() -> Self {
+        let field_elements_csv = include_str!("stark/points.csv");
```

Comment:
It doesn't make much sense to have points in a CSV, just make a Rust file and have an array of FieldElement, or worst case, strings.